.clang-tidy

```diff
@@ -1 +1,2 @@
 Checks: 'google-*'
+WarningsAsErrors: 'clang-analyzer-osx.*'
```

Comment:
Does this file allow comments? If so, we should link to a tracking issue.

Reply:
Per https://clang.llvm.org/extra/clang-tidy/ it looks like this is YAML, so `#` should work for comments.

Reply:
Done.